integration_tests/pom.xml

```diff
@@ -58,39 +58,67 @@
             <classifier>${cuda.version}</classifier>
             <scope>provided</scope>
         </dependency>
+
         <dependency>
             <groupId>com.nvidia</groupId>
-            <artifactId>rapids-4-spark_${scala.binary.version}</artifactId>
+            <artifactId>rapids-4-spark-sql_${scala.binary.version}</artifactId>
```

Comment:
The integration tests **must not** depend on the rapids-4-spark-sql artifact.  The whole point of having tests outside of the rapids-4-spark-sql project is to test against only the dist artifact jar (i.e.: what users will use), and pulling in this jar is like pulling in the aggregator jars -- it defeats the purpose of separating the tests into a different project and does not test against the artifact that will be shipped (i.e.: is not a true integration test).